prometheus/prometheus.go

```diff
@@ -0,0 +1,677 @@
+package prometheus
```

Comment:
From where is this code comming from? Do we need all of this functionalities?

Reply:
It comes from the metric package in zkevm-node. We just need counters now, but if in the future we need any other kind of metric, it will be already there ready to be used.